src/promptflow/promptflow/_constants.py

```diff
@@ -51,7 +51,7 @@ class AvailableIDE:
 _ENV_PF_INSTALLER = "PF_INSTALLER"
 
 # trace related
-TRACE_SESSION_ID_ENV_VAR = "PF_TRACE_SESSION_ID"
+TRACE_SESSION_ID_OP_CTX_NAME = "PF_TRACE_SESSION_ID"
```

Comment:
I remember the operation context name has a special format, maybe "pf-trace-session-id"?
@liucheng-ms @thy09 @lumoslnt

Reply:
Suggest using lowercase characters.
pf_trace_session_id

Reply:
good catch, updated to `pf_trace_session_id`